api/api.go

```diff
@@ -379,6 +379,7 @@ func Provisioners(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
Provisioner responses do not contain sensitive information.

Reply:
I decided to add the header to the provisioner responses because I saw that a few of the provisioners (or at least the ones I checked) contain webhooks nested inside. Taking the AWS provisioner in `authority/provisioners/aws.go` as an example, it's `ctl` field of type `Controller` has a field `webhooks` that stores a list of webhooks.

And regarding the provisioner responses over in `authority/admin/api/provisioners.go`, it looks like the provisioners are represented with a different data type (`linkedca.Provisioner`) there, but it still contains webhooks.

So if I'm understanding correctly the provisioner responses in both packages do contain sensitive information.

Reply:
Removed the header here.